src/include/duckdb/execution/operator/projection/physical_tableinout_function.hpp

```diff
@@ -32,7 +32,7 @@ class PhysicalTableInOutFunction : public PhysicalOperator {
 	                                        OperatorState &state) const override;
 
 	bool ParallelOperator() const override {
-		return true;
+		return !function.with_ordinality;
```

Comment:
I wonder - does ordinality need to block parallel execution? For the table in-out functions, parallel execution is performed over the input rows. It is my understanding that the ordinality is independent for each input row to the table function, and as such no coordination should be required. Am I missing something perhaps?

Reply:
I think this was a result from an earlier discussion on the PR
<https://github.com/duckdb/duckdb/pull/9014#discussion_r1379102490>

Reply:
Aha, but that comment seems to concern the `PhysicalTableFunction` only - whereas this is the `PhysicalTableInOutFunction`, right?